core-errors/src/lib.rs

```diff
@@ -159,6 +159,10 @@ pub enum MessageError {
     /// An error occurs in attempt to charge gas for dispatch stash hold.
     #[display(fmt = "Not enough gas to hold dispatch message")]
     InsufficientGasForDelayedSending = 399,
+
+    /// An error when a syscall is used incorrectly
+    #[display(fmt = "Syscall was used incorrectly")]
+    SyscallUsageError = 400,
```

Comment:
```suggestion
    SyscallUsageError = 311,
```

`InsufficientGasForDelayedSending` is assigned to 399 as to be removed

Moreover, such generic error as `sys-call usage` must be discussed. We have forbidden code 0xffff for such error